scripts/node/twig_functions.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Install Pattern Lab.
+ */
+const shell = require("shelljs");
+const fs = require('fs');
+const path = {
+  'drupalComposer': '../../../../vendor/drupal-pattern-lab',
+  'drupalNoComposer': './../../vendor/drupal-pattern-lab',
+  'standalone': 'vendor/drupal-pattern-lab'
+}
+
+function vendorPathInstalled() {
+  if (fs.existsSync(path.standalone) || fs.existsSync(path.drupalComposer) || fs.existsSync(path.drupalNoComposer)) {
+    return true;
+  }
+  return false;
+}
+
+function vendorDirectoryRoute() {
+  if (fs.existsSync(path.standalone)) {
+    return path.standalone;
+  } else if (fs.existsSync(path.drupalComposer)) {
+    return path.drupalComposer;
+  } else if (fs.existsSync(path.drupalNoComposer)) {
+    return path.drupalNoComposer;
+  } else {
+    return false;
```

Comment:
Instead of returning `false` here maybe consider throwing an error – assuming this is an undesirable state.